src/story.ts

```diff
@@ -259,6 +259,13 @@ export type LoaderFunction<TRenderer extends Renderer = Renderer, TArgs = Args>
   context: StoryContextForLoaders<TRenderer, TArgs>
 ) => Promise<Record<string, any> | void> | Record<string, any> | void;
 
+type Awaitable<T> = T | PromiseLike<T>;
+export type CleanupCallback = () => Awaitable<unknown>;
+
+export type BeforeEach<TRenderer extends Renderer = Renderer, TArgs = Args> = (
```

Comment:
Should this be called BeforeEachFunction to match with other types like LoaderFunction?

Reply:
Oh actually it should be StoryContext.

Reply:
We removed the Function prefix in the public facing API. We didn't do it in this package yet.